Userland/Libraries/LibWebView/ProcessManager.h

```diff
@@ -4,14 +4,15 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#pragma once
```

Comment:
🤔  how did this pass the commit linter originally?

Reply:
Not sure, but it was in https://github.com/SerenityOS/serenity/pull/23745